pyproject.toml

```diff
@@ -2,15 +2,21 @@
 name = "pyper"
 version = "0.1.0"
 description = ""
-authors = ["yansfil <tansfil@gmail.com>"]
+authors = ["yansfil <tansfil@gmail.com>", "ahnsv <ahnsv@bc.edu>"]
+packages = [
+    { include = "pyper" },
+]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = ">=3.6.2,<3.10"
```

Comment:
Looking at tox config, it seems to be applied from python 3.7, how about raising it to 3.7?